### PR TITLE
Support pagination.

### PR DESCRIPTION
**Pull request by darrenfang**

### CODE REVIEW

File: src/Dommel/DommelMapper.cs [hunk 1]
```diff
@@ -868,6 +868,78 @@ private static string BuildSelectSql<TEntity>(Expression<Func<TEntity, bool>> pr
...
+            }
+            else
+            {
+                sql = BuildPaginationSql(connection, tableName, sql, orderBySql, orderByAsc, pageNo.Value, pageSize.Value);
+            }
+
+            return connection.Query<TEntity>(sql, (object) parameters);
+        }
+
+        private static string BuildOrderSql<TEntity>(Expression<Func<TEntity, object>> predicate, bool orderByAsc)
```
Comment (henkmollema) [inline src/Dommel/DommelMapper.cs]: Is it possible to implement this method in the `SqlExpression` class? Seems a better fit to me and doesn't require stuff like `VisitMemberAccess` to be public.

**Comment by DukeCheng** [comment]
@henkmollema Looks the current pagination sql is for SQL Server, not support PostgreSql, Can you redesign this part that can allow more different database, I can contribute for the PostgreSql part.

**Comment by darrenfang** [comment]
@henkmollema Hi, I have refactor the code now.

**Comment by henkmollema** [comment]
This is a breaking change, I will look into merging this into v2.

**Comment by henkmollema** [comment]
This is implemented in v2. Thanks for setting this up!

**Pull request closed without merging**